modules/stats.php

```diff
@@ -50,34 +58,46 @@ function stats_load() {
 
 	add_filter( 'jetpack_xmlrpc_methods', 'stats_xmlrpc_methods' );
 
-
 	add_filter( 'pre_option_db_version', 'stats_ignore_db_version' );
 }
 
 /**
  * Delay conditional for current_user_can to after init.
+ *
+ * @access public
+ * @return void
  */
 function stats_merged_widget_admin_init() {
 	if ( current_user_can( 'view_stats' ) ) {
 		add_action( 'load-index.php', 'stats_enqueue_dashboard_head' );
-		add_action( 'wp_dashboard_setup', 'stats_register_widget_control_callback' ); // hacky but works
+		add_action( 'wp_dashboard_setup', 'stats_register_widget_control_callback' ); // Hacky but works.
 		add_action( 'jetpack_dashboard_widget', 'stats_jetpack_dashboard_widget' );
 	}
 }
 
+/**
+ * Enqueue Stats Dashboard
+ *
+ * @access public
+ * @return void
+ */
 function stats_enqueue_dashboard_head() {
 	add_action( 'admin_head', 'stats_dashboard_head' );
 }
 
 /**
  * Prevent sparkline img requests being redirected to upgrade.php.
  * See wp-admin/admin.php where it checks $wp_db_version.
+ *
+ * @access public
+ * @param mixed $version Version.
+ * @return $version Version.
  */
 function stats_ignore_db_version( $version ) {
 	if (
 		is_admin() &&
-		isset( $_GET['page'] ) && $_GET['page'] == 'stats' &&
-		isset( $_GET['chart'] ) && strpos($_GET['chart'], 'admin-bar-hours') === 0
+		isset( $_GET['page'] ) && 'stats' === $_GET['page'] &&
+		0 === isset( $_GET['chart'] ) && strpos( $_GET['chart'], 'admin-bar-hours' )
```

Comment:
This change will break something. The `isset` [documentation](http://php.net/manual/en/function.isset.php#refsect1-function.isset-returnvalues) says:
> Returns TRUE if var exists and has value other than NULL. FALSE otherwise.

since the `===` operator is used here `(int) 0 =/= (bool) false|true`.

Perhaps you did this to apply yoda notation? Keep in mind that this
```php
isset( $_GET['chart'] ) && strpos($_GET['chart'], 'admin-bar-hours') === 0
```
is not the same than
```php
0 === isset( $_GET['chart'] ) && strpos( $_GET['chart'], 'admin-bar-hours' )
```

cc @kraftbj